sdk/core/Azure.Core/src/Shared/OperationInternalOfT.cs

```diff
@@ -95,7 +95,7 @@ internal class OperationInternal<T> : OperationInternalBase
         ///     Default is <see cref="FixedDelayWithNoJitterStrategy"/>.</param>
         public OperationInternal(IOperation<T> operation,
             ClientDiagnostics clientDiagnostics,
-            Response rawResponse,
+            Response? rawResponse,
```

Comment:
Please add tests for the new logic.   We should be, at minimum, validating that `null` is an accepted value and that the condition for the `InvalidOperationException` works as intended.

Reply:
It is actually tested [here](https://github.com/Azure/azure-sdk-for-net/pull/42686/files#diff-801b4f52618f9b2aad780bfdcc92e92b0f46eda126876c39a9855a22d70fbdf7R148)